map_list/create.tsx

```diff
@@ -35,18 +35,30 @@ export function createMapList(api: Api) {
   const presenter = new MapListPresenter(api, store);
 
   const getRow = (map: PDMap): Row<4> => {
+    const onSelect = action((e: React.MouseEvent<HTMLAnchorElement>) => {
+      if (!store.enableBulkSelect) {
+        return;
+      }
+      e.preventDefault();
```

Comment:
Note: this leads to the undesired UX where hovering over the link will still show it as a link / <a> tag, even though it's not clickable as a link. We should remove the link-ness of it (but somehow without re-rendering the whole table, since that has a big performance cost)